akka-cluster-tools/src/main/resources/reference.conf

```diff
@@ -138,6 +138,9 @@ akka.cluster.singleton {
 }
 
 akka.cluster.singleton-proxy {
+  # The actor name of the singleton actor that is started by the ClusterSingletonManager
+  singleton-name = ${akka.cluster.singleton.singleton-name}
```

Comment:
There is almost never a need to change this name, because it is qualified by the name of singleton manager, which is created by the user with `context.actorOf(..., name = )`, but since it was configurable before and someone might rely on using specific path names I kept it configurable.

Another thing, when you create `ClusterSingletonProxySettings` from a config it is a `Config` with the same layout as defined in reference.conf `akka.cluster.singleton-proxy`. Therefore I wanted to define all settings of the proxy in that section.

What would be possible is to grab the singleton-name from `akka.cluster.singleton.singleton-name` when creating `ClusterSingletonProxySettings` with `def apply(system: ActorSystem): ClusterSingletonProxySettings`